models/comment.go

```diff
@@ -60,6 +69,24 @@ type Comment struct {
 	Content         string `xorm:"TEXT"`
 	RenderedContent string `xorm:"-" json:"-"`
 
+	AddedLabelID   int64  `xorm:"INDEX" json:"-"`
+	AddedLabel     *Label `xorm:"-" json:"-"`
+	RemovedLabelID int64  `xorm:"INDEX" json:"-"`
+	RemovedLabel   *Label `xorm:"-" json:"-"`
+
+	MilestoneID    int64      `xorm:"INDEX" json:"-"`
+	Milestone      *Milestone `xorm:"-" json:"-"`
+	OldMilestoneID int64      `xorm:"INDEX" json:"-"`
+	OldMilestone   *Milestone `xorm:"-" json:"-"`
+
+	AssigneeID    int64 `xorm:"INDEX" json:"-"`
+	Assignee      *User `xorm:"-" json:"-"`
+	OldAssigneeID int64 `xorm:"INDEX" json:"-"`
+	OldAssignee   *User `xorm:"-" json:"-"`
```

Comment:
Any reason we need INDEX for these new "*ID"s?

Reply:
No. I will remove.